### DISCUSSION TITLE
Grafana dashboard for mpp_solar prom output

### DISCUSSION THREAD

**Discussion by cordelster** [jblance/mpp-solar#379] [General]
Hi John, I have created a Dashboard that includes instructions on setup of mpp_solar output to render the dashboard. It's based on the output from LV6548 though I assume that most things PowerMonitor based have similar keys, thus it is a bit portable. It supports standalone and parallel units out of the box. I may, down the road use either the tag variable you have in mpp_solar, or create a new variable  to be able to name multiple sets of systems on the same hardware instance of mpp_solar. 

Would you like me to add  this Dashboard into the git repo and if so how would you prefer the folder layout?
![2023-07-26_18-00](https://github.com/jblance/mpp-solar/assets/4791031/d0a6a02b-fe2f-4bed-a138-85fabccf8416)

**Comment by jblance** [comment]
looks great - thanks

I think a `user_contributed\grafana` folder would be a good location for this
Then a wiki page with details, the above image, and instructions

**Reply by cordelster** [reply]
Very well, I'm still doing some checks, in 12 hrs I'll get this uploaded 'user_contributed\grafana\prometheus' and make sure all the panels are grafana v8 for compatibility. Instructions are in the top row panel, really nothing more than setting some configs for data collection from the inverters, then changing the extension after the file is built for node_exporter.

I'll start off the wiki entry, and add details as questions arise.

**Comment by cordelster** [comment]
Hi John,
I have created a github for the grafana dashboards, and associated files.
I'll be putting more work into this, and it will be more busy, and I thought it better for people to find me for support/issues, etc to track separately.

https://github.com/cordelster/mppsolar_dashboards/

How would you like to handle this on your repo?
Did you want me to leave the code as is, and add a link in the notes or would you like to make it a submodule?
Corey

**Reply by jblance** [reply]
can add something to the readme - and remove the stuff in my repo if its superceded

**Reply by jblance** [reply]
just supply a paragraph or two for me to add to the readme
